splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/Splice_DD_Version.java

```diff
@@ -117,4 +117,15 @@ public long toLong() {
         return majorVersionNumber * 10000000000l + minorVersionNumber * 10000000l + patchVersionNumber * 10000l + sprintVersionNumber;
     }
 
+    static public int compare(Splice_DD_Version version1,Splice_DD_Version version2){
+        long v1=version1.toLong();
+        long v2=version2.toLong();
+
+        if(v1<v2)
+            return -1;
+        else if(v1==v2)
+            return 0;
+        else
+            return 1;
```

Comment:
Nitpick, you can use Long.compare(v1, v2);

Reply:
done